deploy.sh

```diff
@@ -11,31 +9,124 @@ export TAG=1.7-SNAPSHOT
 export DOCKER_REGISTRY_SERVER=docker-registry-default.centralpark2.lightbend.com
 export DOCKER_REGISTRY=$DOCKER_REGISTRY_SERVER/$OPENSHIFT_PROJECT
 
-TOKEN=$1
+## 1. login to the cluster
+login() {
+    echo "Attempting login to Openshift cluster (this will fail on PR builds)"
```

Comment:
btw, why this won't work for PRs?

Reply:
PR jobs don't get travis secrets injected therefor PR jobs won't get `CP2_PLAY_PWD` injected.